prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/jdbc/JdbcService.java

```diff
@@ -0,0 +1,14 @@
+package com.redhat.parodos.tasks.jdbc;
+
+import java.util.List;
+import java.util.Map;
+
+interface JdbcService {
```

Comment:
pls add doc comments to the interface and methods.

Reply:
what do u mean by doc comments?

Reply:
javadoc, but based on the incremental approach, let's skip this in this PR.